.bazelrc.common

```diff
@@ -71,7 +82,6 @@ build:ci-windows-bindist --bes_upload_mode=wait_for_upload_complete --bes_timeou
 # (see https://github.com/buildbuddy-io/buildbuddy/issues/4467)
 # increase the retries as a workaround.
 build:ci-windows-bindist --experimental_build_event_upload_max_retries=256
-build:remote-cache --remote_cache=grpcs://remote.buildbuddy.io
 build:ci-common --remote_timeout=3600
 # Avoid failures of the form `deadline exceeded after 14999958197ns DEADLINE_EXCEEDED`.
 # See https://github.com/tweag/rules_haskell/issues/1498.
```

Comment:
This, and the line below should be redundant now, right?

Reply:
Again, good catch. Apparently, I was asleep when I put this PR together. I did another pass and grouped a few more remote cache flags